cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,52 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import HomePage from '../support/pages/homePage';
+import LaptopsPage from '../support/pages/laptopsPage';
+import ProductPage from '../support/pages/productPage';
+import CartPage from '../support/pages/cartPage';
+import OrderPage from '../support/pages/orderPage';
+import faker from 'faker';
 
-  });
+function generateUserData() {
```

Comment:
for such functions better to create separate support file

Reply:
Done